packages/help-center/src/components/help-center-contact-page.tsx

```diff
@@ -41,11 +41,13 @@ type HelpCenterContactPageProps = {
 	hideHeaders?: boolean;
 	onClick?: () => void;
 	trackEventName?: string;
+	isFreeUser?: boolean;
```

Comment:
I'd use `isUserElegible` instead of `isFreeUser` to avoid confusion since `isUserElegible` !== `isPaidUser` because of some edge cases where a free user can still be eligible for support. This doesn't change the logic, only the wording.